src/integration-test/resources/application-integrationtest.yml

```diff
@@ -0,0 +1,15 @@
+server:
+  port: 8080
+  ssl:
+    key-store: classpath:keystore.jks
```

Comment:
It's here, because Eclipse won't run the makefile and this file will be needed to build the development version of the app.
With the makefile we should simply overwrite it (that's what I missed) with generated one, during the build of the prod version.

Reply:
There are a lot of ways to make sure that these files are there when you need them:
- simply require to run a script before compilation (and mention this in the README)
- use a custom build command in eclipse
- the tests check whether the keystore is there, and if not, then run the needed script
Do not include binary in source.